frontends/web/src/routes/account/send/send.tsx

```diff
@@ -652,9 +652,7 @@ class Send extends Component<Props, State> {
                     <Input
                       label={t('note.title')}
                       labelSection={
-                        <span className={style.labelDescription}>
-                          {t('note.input.description')}
-                        </span>
+                        <OptionalLabel>{t('note.input.description')}</OptionalLabel>
```

Comment:
Should we change this to `lightning.send.description.optional`, or move this to `lightning.description.optional` ?

Reply:
This is not the lightning send but the onchain send component

Reply:
Hah, ok sorry. Didn't check the filename